.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Test jdk15on
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Provide the branch to build and run tests on
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4.1.7
+      with:
+        ref: "${{ github.event.inputs.version }}"
+
+    - name: Set up JDK 8
+      uses: actions/setup-java@v4.2.1
+      with:
+        java-version: '8'
+        distribution: 'temurin'
+
+    - name: Set up Ant
+      uses: cedx/setup-ant@v2
+      with:
+        version: latest
+
+    - name: Verify Java version
+      run: java -version
+
+    - name: Set JDKPATH required for build
+      run: echo "JDKPATH=$JAVA_HOME" >> $GITHUB_ENV
```

Comment:
JDKPATH is an environment variable that needs to be set for the ant build script to run successfully